packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -106,23 +106,20 @@ class _WebViewState extends State<WebView> {
   @override
   void initState() {
     super.initState();
-    _settings = _WebSettings.fromWidget(widget);
+    _updateSettings(_WebSettings.fromWidget(widget));
```

Comment:
This is already sent as part of the platform view creation message (in `creationParams`), no need to send an extra update message again, we should just initialize the WebViewController with the initial settings when we create it.